app/controllers/resources_controller.rb

```diff
@@ -59,20 +59,47 @@ def check_permission
     privilege = params[:privilege]
     raise ArgumentError, "privilege" unless privilege
 
-    result = assumed_role.allowed_to?(privilege, resource)
+    begin
+      result = assumed_role.allowed_to?(privilege, resource)
+
+      write_success_audit(privilege, result)
```

Comment:
please change this to `audit_success` to match the rest of the places we do this